public/app/features/scenes/components/Scene.tsx

```diff
@@ -35,7 +36,15 @@ export class Scene extends SceneObjectBase<SceneState> {
 }
 
 function SceneRenderer({ model }: SceneComponentProps<Scene>) {
-  const { title, layout, actions = [], isEditing, $editor, subMenu } = model.useState();
+  const { title, layout, actions = [], isEditing, $editor, subMenu, standalone } = model.useState();
+
+  if (!standalone) {
+    return (
+      <div style={{ flexGrow: 1, display: 'flex', gap: '8px', overflow: 'auto', minHeight: '100%' }}>
+        <layout.Component model={layout} isEditing={isEditing} />
+      </div>
```

Comment:
this is missing the subMenu and toolbar actions, question is what to do with those. toolbar actions might not be needed. 

subMenu (variables, time picker) could be useful.

Reply:
yeah yeah, that's in no